src/backends/ilp-quoter/index.js

```diff
@@ -24,6 +24,7 @@ class ILPQuoter {
     this.currencyPairs = this.currencyWithLedgerPairs.map((p) => [p[0].slice(0, 3),
                                                                   p[1].slice(0, 3)])
     this.backendUri = opts.backendUri
+    this.backendStatus = 'KO'
```

Comment:
Well, this is in the ctor. Initially status is `KO` and becomes `OK` when the quoter has successful responded. Maybe `KO` isn't the best choice -- what would you use?